bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -166,6 +166,12 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <calc_human_stress_indices phys="clm5_0" >FAST</calc_human_stress_indices>
 <calc_human_stress_indices phys="clm4_5" >NONE</calc_human_stress_indices>
 
+<!--                                                                              -->
+<!-- X. Li [dev.04]                                                               -->
+<!-- Default urban air conditioning flux and building interior temperature scheme -->
```

Comment:
I've made changes to the default values of `urban_explicit_ac` as suggested. I have also made sure the code is backward compatible by making changes to UrbanTimeVarType.F90 and only read in `p_ac` when `urban_explicit_ac = .true.`, and pointing the namelist variable `stream_fldfilename_urbantv` to different urbantv files depending on `urban_explicit_ac`. The only remaining issue is that when I set up a case with `urban_explicit_ac = .true.`, `stream_fldfilename_urbantv` still defaults to the original urbantv file (the same one as `urban_explicit_ac = .false.`). If I did not make a mistake with the logic in namelist_defaults_ctsm.xml, I suspect it might be because the new urbantv file (the one with `p_ac`) is located in my work directory, which the model is not able to read in as defaults. Is this possible?

Reply:
@cathyxinchangli , I just pushed a change to get the correct urbantv streams file.  It worked for me in a CTSM5.1 case.  Please check if it fixes this problem for you.

Reply:
Hi @olyson, yes it worked on my end too! I was able to create a new case where the urbantv streams file is default to the one in my work directory under `urban_explicit_ac = .true.`. The case also ran without an issue. Thank you so much Keith!